cpmpy/solvers/ortools.py

```diff
@@ -464,8 +464,15 @@ def _post_constraint(self, cpm_expr, reifiable=False):
                     return self.ort_model.AddDivisionEquality(ortrhs, *self.solver_vars(lhs.args))
                 elif lhs.name == 'element':
                     # arr[idx]==rvar (arr=arg0,idx=arg1), ort: (idx,arr,target)
-                    return self.ort_model.AddElement(self.solver_var(lhs.args[1]),
-                                                     self.solver_vars(lhs.args[0]), ortrhs)
+                    arr, *idx = lhs.args
+                    if len(idx) > 1: # multi-dim, convert here
```

Comment:
can this reuse the `to_1d_element` function, so it detects that it requires higher D, which is not natively supported, so it calls to_1d, and then continues to post this?